models/staging/commissioning/sus/stg_ecds_consolidated.sql

```
@@ -0,0 +1,312 @@
+
+FROM {{ref('raw_sus_ecds_emergency_care')}} att 
+
+LEFT JOIN {{ref('raw_ukhfd_dim_ref_dates')}} dt 
+ON att."ATTENDANCE_ARRIVAL_DATE" = dt.full_date 
+
+LEFT JOIN {{ref('raw_sus_ecds_attendance_referred_to')}} AS ar 
+ON att."PRIMARYKEY_ID" = ar."PRIMARYKEY_ID"  
+AND "REFERRED_TO_ID" = 1 
+
+LEFT JOIN (SELECT
+				*
```

SQLFluff (4.2.2)

[error] Select targets should be on a new line unless there is only one select target.

(LT09)